build_conan.py

```diff
@@ -49,8 +49,9 @@ def no_conan_profile () :
     pass # do nothing
 
   # TODO: check for failure
-  subprocess.run([CONAN, 'config', 'install', './conan-config'], shell=True,
-                  stdout=subprocess.DEVNULL,
+  # might want to log this?
+  subprocess.run([CONAN, 'config', 'install', f'{os.getcwd()}/conan-config']
```

Comment:
Ya - I think we need to at least check for failure here. 

I think as long as the error goes to stdout - then when `slm` runs the task - it will dump the task output.

Reply:
Nevermind I think it throws an exception on error code. So the script will fail regardless.